src/main/java/com/sumindaa/demo/employeeservice/model/Employee.java

```diff
@@ -42,12 +43,12 @@ public void setLastName(String lastName) {
         this.lastName = lastName;
     }
 
-   // public Date getDob() {
-      //  return dob;
+    // public Date getDob() {
+    //  return dob;
     //}
 
     //public void setDob(Date dob) {
-     //   this.dob = dob;
+    //   this.dob = dob;
     //}
```

Comment:
why

Reply:
earlier added but didn't implement yet. if it will wanted I will implement in future